geomesa-tools/bin/common-functions.sh

```diff
@@ -52,26 +52,36 @@ function findJars() {
 }
 
 #############################################################
-# Download a list of urls to a destination - takes two args:
+# Download a list of urls to a destination - takes three args:
 #  - destination directory
 #  - list of urls passed as an array reference
-# 
+#  - --noPrompt [optional] bypass confirmation. Useful when
+#    you'd like to provide your own confirmation
+#    messages/handling
+#
 # for example:
 #   urls=("1" "2" "3")
 #   downloadURLS /tmp/foobar urls[@]
+# or
+#   downloadURLS /tmp/foobar urls[@] --noPrompt
```

Comment:
could we change ``--noPrompt`` to ``--no-prompt``? seems more consistent...

Reply:
Changed